debian/changelog

```diff
@@ -1,3 +1,10 @@
+containerd.io (1.7.25-1) release; urgency=medium
+
+  * Update containerd binary to v1.7.25
+  * Update runc binary to v1.2.4
```

Comment:
As it's a packaging change, perhaps add a mention of https://github.com/docker/containerd-packaging/pull/403

Reply:
Oh! Sorry that's for the RPM packages only, so in the one below 🙈

Reply:
Ah, good point, updated